packages/g/gtkhash/pspec_x86_64.xml

```diff
@@ -3,8 +3,8 @@
         <Name>gtkhash</Name>
         <Homepage>https://github.com/tristanheaven/gtkhash</Homepage>
```

Comment:
New homepage link is not updated because there are two homepage keys in the package.yml file.